fdbclient/ManagementAPI.actor.cpp

```diff
@@ -2921,52 +2921,53 @@ ACTOR Future<Void> submitBulkLoadTask(Database cx, BulkLoadState bulkLoadTask) {
 
 // Get bulk load task metadata with range and taskId and phase selector
 // Throw error if the task is outdated or the task is not in any input phase at the tr read version
-ACTOR Future<BulkLoadState> getBulkLoadTask(Transaction* tr,
-                                            KeyRange range,
-                                            UID taskId,
-                                            std::vector<BulkLoadPhase> phases) {
-	state BulkLoadState bulkLoadState;
+ACTOR Future<BulkLoadTaskState> getBulkLoadTask(Transaction* tr,
+                                                KeyRange range,
+                                                UID taskId,
+                                                std::vector<BulkLoadPhase> phases) {
+	state BulkLoadTaskState bulkLoadTaskState;
```

Comment:
Do we have a differently named task for bulk dump and bulk load? Or are they both named bulkLoadTaskState whether load or dump?

Reply:
Here are important metadata: (1) BulkLoadTaskState; (2) BulkDumpState; (3) BulkLoadManifest. BulkLoadTaskState is only used for bulkload core engine which persists the metadata for each unit bulkload range (aka. task). BulkDumpState is used for bulk dumping. BulkLoadManifest is the metadata for persisting core information of the load/dumped range. In the next PR, there will be a BulkLoadState metadata which is used for the bulkload job. (aka loading a folder).